.github/workflows/build-test-linux.yml

```diff
@@ -61,7 +48,10 @@ jobs:
     - name: Cache build output
       uses: actions/cache@v2
       with:
-        path: ${{ matrix.cachepath }}
+        path: |
+              ~/.cache/go-build
+              ~/go/pkg/mod
+              /home/runner/.cache/go-build
         key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
         restore-keys: |
```

Comment:
We don't need the restore keys since it would cache the old dependencies package.